src/index.css

```diff
@@ -115,16 +114,17 @@ mark,
 /* Slide up */
 
 .blog-post-anchor {
-  color: #343139;
+  /* color: #343139; */
```

Comment:
Could you remove this line too?

Reply:
I kept these commented because they represent the old colour palette, in case you want to keep it.